packages/xo-web/src/xo-app/vm/tab-console.js

```diff
@@ -53,6 +57,34 @@ export default class TabConsole extends Component {
     this.setState({ minimalLayout: !this.state.minimalLayout })
   }
 
+  _showFormModal = async () => {
+    const commands = await form({
+      defaultValue: '',
+      header: (
+        <span>
+          <Icon icon='file-text' /> {_('copyPasteCommands')}
+        </span>
+      ),
+      render: props => (
+        <Container>
+          <SingleLineRow>
+            <Col size={2}>{_('commands')}</Col>
+            <Col size={10}>
+              <textarea
+                {...props}
+                className='form-control'
+                rows={10}
+                value={props.value}
+              />
+            </Col>
+          </SingleLineRow>
+        </Container>
+      ),
+      size: 'large',
+    })
+    this.refs.noVnc.setClipboard(commands)
```

Comment:
Set the state with the new value to keep the state consistent with the clipboard content.